request-plan9.go

```diff
@@ -2,12 +2,41 @@
 
 package sftp
 
-import "syscall"
+import (
+	"fmt"
+	"syscall"
 
-func fakeFileInfoSys() interface{} {
-	return &syscall.Dir{}
-}
+	sshfx "github.com/pkg/sftp/internal/encoding/ssh/filexfer"
+)
 
 func testOsSys(sys interface{}) error {
```

Comment:
Ah, this can likely borrow from the fixes I already pushed for the `Attributes` converting code…